lib/plausible/stats/query.ex

```diff
@@ -7,16 +7,23 @@ defmodule Plausible.Stats.Query do
             filters: %{},
             sample_threshold: 20_000_000,
             imported_data_requested: false,
-            include_imported: false
+            include_imported: false,
+            now: nil
 
   require OpenTelemetry.Tracer, as: Tracer
   alias Plausible.Stats.{FilterParser, Interval}
 
   @type t :: %__MODULE__{}
 
   def from(site, params) do
+    now =
+      NaiveDateTime.utc_now()
+      |> NaiveDateTime.truncate(:second)
```

Comment:
since Elixir 1.15 you can also:

```suggestion
    now = NaiveDateTime.utc_now(:second)
```

Reply:
Aw thanks!